File: codekata/Basics/square_or_not.py
N, M=[int(x) for x in input().split()]
n=N*M
sum = 0
for i in range(1, N+1):
    n1=i**2
    if (n1 == n):
        sum=sum+1
if(sum>0):
    print("yes",end="")
else:
    print("no",end="")

File: javascript/codekata/basics/square_or_not.js
const readline=require('readline');
const inp=readline.createInterface({
  input:process.stdin});
const n=[];
inp.on("line",(data)=>{
  n.push(data);});
inp.on("close",()=>{
  var a=[],s,c=0;
  a=n[0].split(" ");
  var N=parseInt(a[0]),M=parseInt(a[1]);
  var p=N*M;
  for(var i=1;i<=N;i++){
    s=Math.pow(i,2);
    if(s==p)
      c+=1;
  }
  if(c==1)
    console.log("yes");
  else
    console.log("no");
});
© 2020 GitHub, Inc.
